src/ActionModal/utils/networkMessages/polkadot-testnet.js

```diff
@@ -1,4 +1,19 @@
-import { ApiPromise } from "@polkadot/api"
+import { ApiPromise, WsProvider } from "@polkadot/api"
+
+// will only be inited once per session
+let api
+async function getAPI() {
+  if (api) {
+    return api
+  }
+  api = new ApiPromise({
+    provider: new WsProvider(
+      "wss://kusama-rpc.polkastats.io/apikey/HunRG7eUwMTjaBAkz1A6GU1MqBJapaYsYfmU5EVzpAebMr8/"
```

Comment:
APIkey should be `ENV` variable?

Reply:
whole thing should just be in the api. will need to consider rotating the apikey @mariopino

Reply:
No problem as that was only for testing and I'll remove the exposed RPC, now we are using parity RPC.

btw tested and works awesome @faboweb!!